SPECS/cpulimit/cpulimit.spec

```
@@ -0,0 +1,159 @@
+# vim: set sw=4 ts=4 et nu:
+#
+# spec file for package cpulimit
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+Name:           cpulimit
+Version:        2.8
+Release:        2%{?dist}
+Summary:        Limit the CPU Usage of a Process
+License:        GPL-2.0-or-later
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Monitoring
+URL:            http://limitcpu.sourceforge.net/
+Source0:        https://downloads.sourceforge.net/limitcpu/%{name}-%{version}.tar.gz
+Patch0:         %{name}-2.2-do_not_forget_version.patch
```

NIT: Given the patch does `2.8` and not `2.2`
```suggestion
Patch0:         %{name}-2.8-do_not_forget_version.patch
```